### DISCUSSION TITLE
Curso de Introducción al Desarrollo Web: HTML y CSS *Gratis*

### DISCUSSION THREAD

**Discussion by oznema** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#330] [General]
Hola, compañeros explorers. 

El post es solo para compartirles el curso gratis con certificación de Google:
Curso de Introducción al Desarrollo Web: HTML y CSS
https://learndigital.withgoogle.com/activate/course/web-development-I?enroll-success=1

Esta seccionado en 2 partes, espero les sea de utilidad.

**Comment by JuanSalvador73** [comment]
Gracias por el aporte :D
